modules/web/server/src/main/scala/observe/web/server/http4s/ObserveCommandRoutes.scala

```diff
@@ -71,11 +73,20 @@ class ObserveCommandRoutes[F[_]: Async: Compression](
     //     Ok(s"Cancel Pause sequence $obsId")
     //
     case req @ POST -> Root / ObsIdVar(obsId) / StepIdVar(stepId) / ClientIDVar(clientId) /
-        "breakpoint" / ObserverVar(obs) / BooleanVar(bp) =>
+        "breakpoint" / ObserverVar(obs) / BreakpointVar(bp) =>
       ssoClient.require(req) { user =>
-        oe.setBreakpoint(obsId, user, obs, stepId, bp) *>
+        oe.setBreakpoints(obsId, user, obs, List(stepId), bp) *>
           NoContent()
       }
+
+    case req @ POST -> Root / ObsIdVar(obsId) / ClientIDVar(clientId) /
+        "breakpoints" / ObserverVar(obs) / BreakpointVar(bp) =>
+      ssoClient.require(req) { user =>
+        req.decode[List[Step.Id]] { steps =>
```

Comment:
How is this decoded? Just a comma-separated list of step ids?

Reply:
Yes like `["s-f5efec44-32f4-3b6b-8e9a-39dcf55ed31c", "s-800d515f-835d-3522-aca3-1f21b2865d84"]`